tests/xtro-sharpie/iOS-HomeKit.todo

```diff
@@ -1,4 +1 @@
-!missing-selector! HMAccessory::supportsIdentify not bound
-!missing-selector! HMAccessorySetupPayload::initWithURL: not bound
-!missing-selector! HMHome::addAndSetupAccessoriesWithPayload:completionHandler: not bound
-!missing-type! HMAccessorySetupPayload not bound
+
```

Comment:
Maybe remove those `.todo` files if empty in the future.

Reply:
@VincentDondain I followed the pattern I saw in diff PRs. Since the process is new, I'm not quite sure what we should do (although removing them seems ok in my side). Lets raise this as a comment in the weekly call and that way we are all in the same page.

Reply:
I asked Sebastien when I faced that question:
https://xamarinhq.slack.com/archives/C03CCJNR7/p1516915121000352

Reply:
> not mandatory (to work) but yes (for hygiene)

It's fine for this PR but you'll want to remove them in the future.

Reply:
@VincentDondain I'll do it now, then merge. No harm in doing it ASAP.